Library/Bookapp/views.py

```diff
@@ -0,0 +1,76 @@
+from django.shortcuts import render
+from django.contrib.auth import logout, login
+from django.contrib.auth.decorators import login_required
+from Bookapp.serializers import *
+from rest_framework.views import APIView
+from rest_framework.response import Response
+from rest_framework import status
+from .models import Book,User
+
+
+class CreateUserView(APIView):
+
+    def post(self,request):
+        serializer=UserSerializer(data=request.data)
+        if serializer.is_valid():
+            user=serializer.save()
+            return Response({'status':'Created the user'})
+        return Response({'error':serializer.errors})
+
+
+class LoginView(APIView):
+
+    def post(self,request):
+        user_id=request.data.user_id
+        user=User.Objects.get(pk=user_id)
```

Comment:
we need to do authentication part here. I will teach you that today.

Reply:
I added the authentication part sir